setup.py

```diff
@@ -31,7 +31,13 @@
 
 install_requires = [
     'amqp>=1.4.9,<2.0',
+    'Flask>=0.11.1',
+    'Flask-Admin>=1.4.2',
+    'Flask-Breadcrumbs>=0.3.0',
+    'Flask-Script>=2.0.5',
+    'Flask-CLI>=0.4.0',
```

Comment:
It's required by a lot of other packages anyway, `invenio-search` included: https://github.com/inveniosoftware/invenio-search/blob/0fc8f760fa3f5abb47e02eaa70735778cdc9fc3f/setup.py#L71